src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/collections/TimeExpiringSet.java

```diff
@@ -0,0 +1,76 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.collections;
```

Comment:
License information missing. Please take care to update the license to use 2020 as the year.

Reply:
Added License info to all files in this PR